pkg/trace/api/profiles_test.go

```diff
@@ -119,8 +167,38 @@ func TestProfileProxyHandler(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-		if !strings.Contains(string(slurp), "misconfigured") {
+		if !strings.Contains(string(slurp), "invalid intake URL") {
 			t.Fatalf("invalid message: %q", slurp)
 		}
 	})
+
+	t.Run("multiple_targets", func(t *testing.T) {
+		called := make(map[string]bool)
+		handler := func(w http.ResponseWriter, req *http.Request) {
+			called[fmt.Sprintf("%s|%s", req.Host, req.Header.Get("DD-API-KEY"))] = true
+		}
+		srv1 := httptest.NewServer(http.HandlerFunc(handler))
+		srv2 := httptest.NewServer(http.HandlerFunc(handler))
+
+		additionalEndpoints := make(map[string][]string)
+		additionalEndpoints[srv2.URL] = []string{"dummy_api_key_1", "dummy_api_key_2"}
+		// this should be ignored
+		additionalEndpoints["foobar"] = []string{"invalid_url"}
+		defer mockConfigMap(map[string]interface{}{
+			"apm_config.profiling_dd_url":               srv1.URL,
+			"apm_config.profiling_additional_endpoints": additionalEndpoints,
```

Comment:
```suggestion
			"apm_config.profiling_additional_endpoints": map[string][]string{
			    srv2.URL: []string{"dummy_api_key_1", "dummy_api_key_2"},
			    "foobar": []string{"invalid_url"},
			},
```
We can drop the odd declaration above. It's not needed or used anywhere.